### PR TITLE
Updated sphinx to v0.2.7

### PR DESCRIPTION
**Pull request by gonzaloaune**
Update Sphinx to 0.2.7

### CODE REVIEW

**Comment by louneskmt** [comment]
Hey, thanks for this PR! Have you tested this release? We did for the v0.2.6 but we had to revert the update as there were a bug in the Connect UI and the connection details were not displayed.

**Comment by gonzaloaune** [comment]
@louneskmt yes we tested it and works as expected!

**Comment by louneskmt** [comment]
Awesome, I'll give it a try tomorrow!

File: apps/sphinx-relay/docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ x-logging: &default-logging
 services:
   sphinx-relay:
     container_name: sphinx-relay
-    image: sphinxlightning/sphinx-relay:v2.0.3@sha256:9864925add480da27f04b82226d4dc0144c99941fae8c8e6e748c9705ec3bd36
+    image: sphinxlightning/sphinx-relay:v2.0.7@sha256:e20873740ad910f1417cb8327a90e10278c8ac8c3ca22b9d1e5ff9c55b232f27
```
Comment (lukechilds) [inline apps/sphinx-relay/docker-compose.yml:11]: Where did you get this digest from?

The digest I'm getting is `54b16d19786ab215b231d1cb7dd4c3ead55fe02a437c52493086716f5d858a98`. Did you maybe pin to a hash of an individual image, and not the multi-arch manifest? If so it means all architectures will pin down the same single arch image and it'll break on either x86 or ARM.

You can just `docker pull` to get the digest

```
$ docker pull sphinxlightning/sphinx-relay:v2.0.7
v2.0.7: Pulling from sphinxlightning/sphinx-relay
Digest: sha256:54b16d19786ab215b231d1cb7dd4c3ead55fe02a437c52493086716f5d858a98
Status: Image is up to date for sphinxlightning/sphinx-relay:v2.0.7
docker.io/sphinxlightning/sphinx-relay:v2.0.7
```

Comment (gonzaloaune) [reply apps/sphinx-relay/docker-compose.yml]: That's exactly what happened! My bad the digest is fixed now, will be more careful in the future.

**Comment by lukechilds** [comment]
Replaced by https://github.com/getumbrel/umbrel/pull/731

**Pull request closed without merging**